asciidoc-confluence-publisher-client/src/it/java/org/sahli/asciidoc/confluence/publisher/client/ConfluencePublisherIntegrationTest.java

```diff
@@ -112,7 +112,7 @@ public void publish_sameAttachmentsPublishedMultipleTimes_publishProcessDoesNotF
         String title = uniqueTitle("Single Page");
         Map<String, String> attachments = new HashMap<>();
         attachments.put("attachmentOne.txt", absolutePathTo("attachments/attachmentOne.txt"));
-        attachments.put("attachmentTwo.txt", absolutePathTo("attachments/attachmentTwo.txt"));
+        attachments.put("это 太陽 მზე!", absolutePathTo("attachments/это 太陽 მზე!.txt"));
```

Comment:
Since there is now a separate test for the non-ascii title for attachments, you could revert the changes in this test method again to only use attachment one and two. Thank you!

Reply:
No, this is actually the scenario "publish_sameAttachmentsPublishedMultipleTimes" that fails with non-ASCII symbols in attachments names. The test modified to support this case.

Reply:
Not sure whether I understand your comment correctly. My understanding is that the new test case `publish_attachmentWithNonAsciiCharactersInTitle_uploadsAttachmentWithCorrectName` already makes sure that the Confluence Publisher can correctly deal with attachments with non-ascii characters in title/filename, so there is no need to extend any other existing test. It might well be that this test here (and probably many more) fails if there is an attachment with non-ascii characters and the support is not implemented, but that for me, it is all about test isolation.

Reply:
The original issue I found, that scenario 'sameAttachmentsPublishedMultipleTimes' fails when use non-ascii characters in filename. So I modified it. Possibly it can be split on 2 cases:
* 'publish_sameAttachmentsWithNonAsciiNamesPublishedMultipleTimes_publishProcessDoesNotFail' 
* 'publish_sameAttachmentsWithAsciiNamesPublishedMultipleTimes_publishProcessDoesNotFail'
IMHO it's extra complexity.